data/faq/es/home.yaml

```diff
@@ -1,3 +1,12 @@
+- question: ¿Es SimpleLogin un proveedor de correo desechable?
```

Comment:
ah don't worry about translations, we have a process of translation in house that can take care of that :)